### ISSUE TITLE
Error creatring an inventory item. Product catalog description must not contain the brand

### ISSUE THREAD

**Issue by Norbbs** [fjborquez/katara#96]
### Bug Description
Adding a new item in the inventory table, when I need to add a new product to catalog because it doesn't exist, the app adds it but the description contains the brand and it is wrong, because our business rul says the product catalog description = product type + product presentation.

### Steps to Reproduce
1. Go to inventory
2. Click on "add" link 
![Image](https://github.com/user-attachments/assets/0764c37f-2fa4-41f9-9800-5c2cdfdcd278)
3. Click on "add new product"
![Image](https://github.com/user-attachments/assets/76c0e1ea-b7f3-4351-98ab-1d44432e1ff6)
4. Click on "create" button.
5. The app goes back to "add product to inventory form"
6. El field "product" is empty 
7. Then search in the field "product"![Image](https://github.com/user-attachments/assets/65a4e62c-59b2-4c71-b936-a1aab6c66ab1)
7. The brand Quirch appears in the description and IT IS RIGHT, but JUST FOR VISUAL, it doesnt mean the system has to save it.
![Image](https://github.com/user-attachments/assets/8f953fb8-ef38-4892-a98a-84598157548d)
9. Click on "create" button.
10. Inventory created sucessfully.
![Image](https://github.com/user-attachments/assets/1754ad82-dce8-46fe-b801-e63c96509d9f)
11. But the "trutro" product doesnt appear in the inventory.
12. Refresh the browser.
![Image](https://github.com/user-attachments/assets/1e291714-7389-415c-a05c-d038c7ee3849)

### Expected Behavior
1. When user adds a new product to inventory, it necessary the catalog description just save the product type + presentation.
2. When user adds a new product to catalog and back to the "add product to inventory" form the product field must be filled up with the product just created in the catalog.
3. When user adds a new product to inventory, it necessary the inventory table refresh automatically.  

### Impact

_The current bug significantly affects the user experience and operational efficiency. If the product catalog descriptions are saved incorrectly, it can lead to confusion among users regarding product identification and selection._

**Comment by Norbbs** [comment]
@fjborquez

**Comment by fjborquez** [comment]
1. done

**Comment by Norbbs** [comment]
> 3. El producto no aparece porque aún está siendo procesado por azula-thunder, entonces no tiene ningun estado y los productos sin estados no deben aparecer.

@fjborquez Ok, pero hay una manera de hacer que si aparezca ?

Sino, preferiría que no regrese a la tabla de inventario después de crear sino que se quede en el mismo formulario "reseteado" por si quiere agregar otro y después si el usuario quiere volver a la tabla, que regrese pero que se vea el nuevo item recién agregado...

**Comment by fjborquez** [comment]
Claro, esperando que el proceso termine o quitando la restricción de que no aparezcan los que no tienen estado.

**Comment by Norbbs** [comment]
> Claro, esperando que el proceso termine o quitando la restricción de que no aparezcan los que no tienen estado.

Ok @fjborquez .. Hagamos esto mientras: después de agregar un nuevo item al inventario que se mantenga en el formario, este se debe resetear para que el usuario tenga todos los campos limpios para que agregue otro producto si así lo desea y que le salga el mensaje de exito de product added.

**Comment by fjborquez** [comment]
3. done
